src/brogue/Recordings.c

```diff
@@ -1212,11 +1213,10 @@ void saveGame() {
     getAvailableFilePath(filePathWithoutSuffix, defaultPath, GAME_SUFFIX);
     filePath[0] = '\0';
 
-    deleteMessages();
     do {
         askAgain = false;
         if (getInputTextString(filePathWithoutSuffix, "Save game as (<esc> to cancel): ",
-                               BROGUE_FILENAME_MAX - strlen(GAME_SUFFIX), filePathWithoutSuffix, GAME_SUFFIX, TEXT_INPUT_FILENAME, false)) {
+                               MAX_TEXT_INPUT_FILENAME_LENGTH, filePathWithoutSuffix, GAME_SUFFIX, TEXT_INPUT_FILENAME, true)) {
```

Comment:
Q: personally I like the prompt matching the UI of the messages on the post-game screen (I put effort in the past to preserve this), is there any functionality downside to leaving this one false?

Reply:
i'm not sure what you mean by the "prompt matching the UI of the messages".

the genesis of this change is from the feats/achievements rework (which is pretty much done). i'll try to recall my thinking as it relates to the save dialog. in my current implementation, feats have a score value, so that required shifting the list of achievements to the left. that resulted in the save dialog looking out of place but getInputTextString doesn't allow for granular placement of the prompt. also, i wasn't sure how many feats might need to be displayed so i thought it might be better to move the save to a new screen entirely.

it might be worth doing a quick discord call to talk through (with screen share).